contracts/randdrop/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "randdrop"
+version = "0.1.0"
+authors = ["Katarina","Simon"]
+edition = "2021"
+publish = false
+license = "Apache-2.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# for quicker tests, cargo test --lib
+# for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces"]
+
+[dependencies]
+nois = "0.7.0"
+
+cosmwasm-std = { version = "1.2.3" }
+cosmwasm-schema = { version = "1.2.3" }
+cw-storage-plus = { version = "1.1.0" }
+serde = { version = "1.0.103", default-features = false, features = ["derive"] }
```

Comment:
This could be a dev-dependency since you are not using it directly in contract code. But it does not matter much.